xml/admin_monitoring_alerting.xml

```diff
@@ -366,36 +366,102 @@ placement:
     </para>
   </sect1>
 
-  <!--  <sect1>
+  <sect1>
    <title>SNMP Trap Receiver</title>
    <para>
     If you want to get notified about &prometheus; alerts via SNMP traps, then
     you can install the &prometheus; &alertmanager; SNMP trap receiver via
-    &cephadm;. To do so, you need to enable it in the Pillar under the
-    <option>monitoring:alertmanager_receiver_snmp:enabled</option> key. The
-    configuration of the receiver must be set under the
-    <option>monitoring:alertmanager_receiver_snmp:config</option> key.
+    &cephadm;. To do so, you need to create a service and placement
+    specification file with the following content:
    </para>
-   <example>
-    <title>SNMP Trap Configuration</title>
+   <note>
+     <para>
+       For more information on service and placement files, see
+       <xref linkend="cephadm-service-and-placement-specs"/>.
+     </para>
+   </note>
+<screen>
+service_type: container
+service_id: prometheus-webhook-snmp
+placement:
+    <ADD_PLACEMENT_HERE>
+image: registry.opensuse.org/opensuse/prometheus-webhook-snmp:latest
+args:
```

Comment:
Need to get proper registry link here - @votdev :)

Reply:
This should be `registry.suse.com/ses/7/prometheus-webhook-snmp:latest`, but we had to wait for SES7 M18 to validate this.